Applications/ConsoleReferenceServer/Quickstarts.ReferenceServer.Config.xml

```diff
@@ -112,7 +114,43 @@
         <SecurityMode>SignAndEncrypt_3</SecurityMode>
         <SecurityPolicyUri></SecurityPolicyUri>
```

Comment:
expectation were that here are add the ECC profiles if the ECC cert is configured as app cert.